l10n_nl_state/data/l10n_nl_states.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <data noupdate="1">
```

Comment:
Hoi sluit voor de codes e.d. aan bij de ISO standaard: https://en.wikipedia.org/wiki/ISO_3166-2:NL (doe die link ook maar in de README.rst).

En neem dan gelijk even de zes eilanden en hun codes mee.